seed/views/portfoliomanager.py

```diff
@@ -176,9 +200,14 @@ def get_list_of_report_templates(self):
             raise PMExcept("Could not find rows key in template response; aborting.")
         templates = template_object["rows"]
         template_response = []
-        for t in templates:
+        sorted_templates = sorted(templates, key=lambda x: x['name'])
```

Comment:
Sorting the templates here is also an important addition.  This sorts the "parent" templates, children stay as they are underneath the parents.  This makes the template list dropdown choices look exactly like they do over on ESPM when they are sorted by name there.  SF sorts them by name because they have one parent object with a prefix asterisk, as it is their primary one.  Now that one shows up at the top of the list both on ESPM *and* on this dropdown.  :+1: :+1: